TestCtlV2GetRoleUserWithProxy: User username does not exist

via semaphore (https://semaphoreci.com/coreos/etcd/branches/release-3-2/builds/4)

```
--- FAIL: TestCtlV2GetRoleUserWithProxy (1.18s)
	ctl_v2_test.go:183: failed to get user (read /dev/ptmx: input/output error (expected "User: username", got ["auth: User username does not exist.\r\n"]))
```

--- a/e2e/ctl_v2_test.go
+++ b/e2e/ctl_v2_test.go
@@ -163,7 +163,7 @@ func TestCtlV2GetRoleUserWithProxy(t *testing.T) { testCtlV2GetRoleUser(t, &conf
 func testCtlV2GetRoleUser(t *testing.T, cfg *etcdProcessClusterConfig) {
 	defer testutil.AfterTest(t)
 
-	epc := setupEtcdctlTest(t, cfg, true)
+	epc := setupEtcdctlTest(t, cfg, false)
 	defer func() {
 		if err := epc.Close(); err != nil {
 			t.Fatalf("error closing etcd processes (%v)", err)

